README.md

```diff
@@ -1,57 +1,250 @@
+
+2. Enter the criteria for the phone number you need, and then click Search.
+![Configure Phone Number](https://github.com/ballerina-platform/module-ballerinax-twilio/tree/master/ballerina/resources/phone-number-config.png?raw=true)
+
+- Country: Select the desired country from the drop-down menu.
+- Number or Location: Select the desired option to search by digits/phrases, or a specific City or Region.
+- Capabilities: Select your service needs for this number. 
+
+3. Click Buy to purchase a phone number for your current project or sub-account.
+![Search Results](https://github.com/ballerina-platform/module-ballerinax-twilio/tree/master/ballerina/resources/search-phone-number.png?raw=true)
+> **Notice:** Many countries require identity documentation for Phone Number compliance. Requests to provision phone numbers with these regulations will be required to select or add the required documentation after clicking Buy in Console. To see which countries and phone number types are affected by these requirements, please see twilio's [Phone Number Regulations](https://www.twilio.com/guidelines/regulatory) site.
+
+### Step 3: Obtain a Twilio Account SID with Auth Token.
+Twilio uses two credentials to determine which account an API request is coming from: The Account SID, which acts as a `username`, and the Auth Token which acts as a `password`. You can find your account SID and auth token in your [Twilio console](https://www.twilio.com/console).
+
+![Twilio Credentails](https://github.com/ballerina-platform/module-ballerinax-twilio/tree/master/ballerina/resources/get-credentails.png?raw=true)
+
+Your account's Auth Token is hidden by default. Click show to display the token, and hide to conceal it again. For further information click [here](https://support.twilio.com/hc/en-us/articles/223136027-Auth-Tokens-and-How-to-Change-Them)
+
+## Quickstart
+
+To use the Twilio connector in your Ballerina application, update the `.bal` file as follows:
+
+### Step 1 - Import the package
+
+Import the Twilio package into your Ballerina program as shown below:
+
+```ballerina
+import ballerinax/twilio;
+```
+
+### Step 2 - Create a new connector instance
+
+To create a new connector instance, add a configuration as follows (You can use [configurable variables](https://ballerina.io/learn/by-example/configurable.html) to provide the necessary credentials):
+
+```ballerina
+configurable string accountSID= ?;
+configurable string authToken = ?;
+
+twilio:ConnectionConfig twilioConfig = {
+    auth: {
+        username: accountSID,
+        password: authToken
+    }
+};
+
+twilio:Client twilio = check new (twilioConfig);
+```
+
+### Step 3 - Invoke the connector operation
+
+1. Invoke the connector operation using the client as shown below:
+
+```ballerina
+public function main() returns error? {
+    twilio:Account account = check twilio->fetchAccount(accountSID);
```

Comment:
Can we use a more popular use case (i.e. make a call, send a message) for the quick start sample? :)

Reply:
Sure. Addressed in [e53338e](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/209/commits/e53338ef5bd6bd3cc6640678b571a81183fbdbbf)